### DISCUSSION TITLE
Does the JobRunner Licensed Pro can handle misfired jobs when it comes up ?

### DISCUSSION THREAD

**Discussion by bhushanibm** [jobrunr/jobrunr#519] [General]
The JobRunner is a java library which embeds inside the service docker. This service are running as pod inside the k8 env. There might be certain times when the service is down due to some maintenance and if jobs are scheduled at this point of time are going to be missed. When service comes back to running, will all the jobs which were missed original since the service were down, will it be re-triggerred ?

**Comment by bhushanibm** [comment]
@rdehuyss :- Can you please let us know whether the JobRunner License pro version can it handle the misfires as described above ?

**Comment by rdehuyss** [comment]
Hi @bhushanibm - currently this is not the case but I added it to the JobRunr Pro product backlog. 

This is something I can add in Q3.

**Reply by bhushanibm** [reply]
@rdehuyss :- Is the above functionality added to roadmap of Q3. As an update, I am working for Precisely company and we are in the process of procurement of Pro License. Above functionality is needed for some of the new feature developed on our side and hence this ask.

**Reply by rdehuyss** [reply]
Hi @bhushanibm , it's added to the backlog but i cannot guarantee it will be delivered by then.

For important features that a customer needs swiftly, I use paid feature development and there are already some of those paid features planned.

It will be available somewhere Q4 though.
